traffic_ops/testing/api/config.go

```diff
@@ -0,0 +1,81 @@
+/*
+   Copyright 2015 Comcast Cable Communications Management, LLC
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/kelseyhightower/envconfig"
+)
+
```

Comment:
If you are referring to using "vendor" code from other components, I have not been able to figure that out yet.  But that dependency is only in this tool atm

Reply:
It can, by putting `traffic_ops_golang/config.go` in its own package. So, move to `traffic_ops_golang/config/config.go`, and things like this can `import (toconfig "traffic_ops_golang/config")` 

(I'd name the package `toconfig` directly if anything besides testing frameworks need it, but that seems unlikely.)

Reply:
we should address this later...